services/distribution/src/test/resources/dgc/dcc-validation-rule.json

```diff
@@ -0,0 +1,146 @@
+{
```

Comment:
Same: Seems to be a doublette of the file at services/distribution/src/main/resources/dgc/

Reply:
And the files below also...?